backend/endpoints/rom.py

```diff
@@ -497,6 +497,14 @@ async def delete_roms(
         log.info(f"Deleting {rom.file_name} from database")
         db_rom_handler.delete_rom(id)
 
+        # Update collections to remove the deleted rom
+        collections = db_collection_handler.get_collections_by_rom_id(id)
+        for collection in collections:
+            collection.roms = [rom_id for rom_id in collection.roms if rom_id != id]
+            db_collection_handler.update_collection(
+                collection.id, {"roms": collection.roms}
+            )
```

Comment:
should this instead happen in `db_rom_handler.delete_rom` as a side-effect?

Reply:
hmm idk, i stick to the idea of just interacting with one table per handler, but idk if that's the best approach